### PR TITLE
Release

### PR DESCRIPTION
**Pull request by IvanBorislavovDimitrov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## v3.2.2
+* Fixed random failures during starting operation caused by wrong CSRF token handling
+* Files with same digest but different names will not be reuploaded
+* Fixed filtration by namespace in cf mta command
+* Improved resilience during deploy from URL
+* Fixed enablement of skip-ssl-validation
+* Binaries are now statically linked
```
Comment (boyan-velinov) [inline CHANGELOG.md:7]: Can we explain what was the issuue?
E.g link to https://github.com/cloudfoundry/multiapps-cli-plugin/issues/186 or so

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## v3.2.2
+* Fixed random failures during starting operation caused by wrong CSRF token handling
```
Comment (boyan-velinov) [inline CHANGELOG.md:2]: Can we put here what was the issue, e.g. some 403 returned from backend

Comment (IvanBorislavovDimitrov) [reply CHANGELOG.md:2]: Cannot find a failed job with this error...

File: CHANGELOG.md (hunk 1 above)
Comment (IvanBorislavovDimitrov) [reply CHANGELOG.md]: Done

**Review by boyan-velinov — APPROVED** [review]

**Pull request merged**